### PR TITLE
Cu 86b0kyh2q homepage flow

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
- Main Layout
- Notifications button
  - Notification page
- Sessions
  - Create Session
- Session Summary

The PNGs are just used for placeholders at the moment.

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b0kyh2q Homepage flow](https://app.clickup.com/t/86b0kyh2q)

File: src/components/style/CustomBackdrop.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { BottomSheetBackdropProps } from "@gorhom/bottom-sheet";
```
Comment (whamester) [inline src/components/style/CustomBackdrop.tsx]: this component could be in the ui folder

components/ui/CustomBackdrop

**Review by whamester — APPROVED** [review]

File: src/components/style/CustomBackdrop.tsx (hunk 1 above)
Comment (ReinhardtBotha) [reply src/components/style/CustomBackdrop.tsx]: Done: https://github.com/LangaraBeetles/weaup-react-native-app/pull/24/commits/3beaab4b7c02f88daf5b6efd1ef22b835100e1e4

**Review by Victorportus — APPROVED** [review]
Looks great!

**Review by zeckkimon — APPROVED** [review]
Nice!

I just noticed that starting the session does not set the isTrackingEnabled automatically. But it could be for the next PR related to sessions/homepage.

**Pull request merged**